scipy/stats/tests/test_stats.py

```diff
@@ -379,7 +381,7 @@ def test_pROUNDROUND(self):
         assert_approx_equal(r,1.0)
 
 
-@skip_xp_backends(cpu_only=True)
+@make_skip_xp_backends('pearsonr', xp_capabilities_table)
```

Comment:
Now we only have to apply one decorator to test classes. The `make_skip_xp_backends` decorator factory looks up the information in `xp_capabilities_table['pearsonr']`, creates all the appropriate decorators with that information, then returns a single decorator that chains them all together.

If we were to name test classes in a standard way (which is usually the case right now, especially as we go back through and do arrray API translations), we wouldn't have to provide the function name as an argument.

Reply:
If we defined `xp_capabilities_table` centrally in `_array_api.py` then the decorators could just use it directly?

Reply:
They could, but didn't we want to keep it close to the functions?

Reply:
> I mean just defining the empty dictionary centrally

Ah, yes, thought about that, but shied away given the new multithreading work. I would definitely like that better than separate dictionaries in each module, though. It would mostly eliminate the advantages of attaching the information to the function. Is it ok to have a global dictionary that's mutated all over SciPy (but only once, on import)?